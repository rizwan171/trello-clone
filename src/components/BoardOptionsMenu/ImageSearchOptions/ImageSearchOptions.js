import React from 'react';

const ImageSearchOptions = () => {
  return (
    <div className='flex flex-col'>
      <input
        // value={}
        type="text"
        placeholder='Search..'
        autoFocus
        className='w-full text-left box-border text-sm outline-none border-2 border-trello-gray-200 shadow bg-trello-gray-400 hover:bg-trello-gray-400 rounded-md mb-2 p-2'
      // onBlur={() => setOpen(false)}
      />
      <div className='w-full grid grid-cols-2 gap-1'>
        <div className='bg-black h-28 rounded-md '></div>
        <div className='bg-black h-28 rounded-md'></div>
        <div className='bg-black h-28 rounded-md'></div>
        <div className='bg-black h-28 rounded-md'></div>
      </div>
    </div>
  );
};

export default ImageSearchOptions;
