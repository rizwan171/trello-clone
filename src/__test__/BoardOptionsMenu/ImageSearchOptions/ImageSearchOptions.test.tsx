import TestRenderer from "react-test-renderer";
import ImageSearchOptions from "../../../components/BoardOptionsMenu/ImageSearchOptions/ImageSearchOptions";

describe("ImageSearchOptions", () => {
  it("should render successfully", () => {
    const component = TestRenderer.create(<ImageSearchOptions />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
