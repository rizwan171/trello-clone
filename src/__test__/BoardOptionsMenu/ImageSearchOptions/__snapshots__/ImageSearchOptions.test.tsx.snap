// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageSearchOptions should render successfully 1`] = `
<div
  className="flex flex-col"
>
  <input
    autoFocus={true}
    className="w-full text-left box-border text-sm outline-none border-2 border-trello-gray-200 shadow bg-trello-gray-400 hover:bg-trello-gray-400 rounded-md mb-2 p-2"
    placeholder="Search.."
    type="text"
  />
  <div
    className="w-full grid grid-cols-2 gap-1"
  >
    <div
      className="bg-black h-28 rounded-md "
    />
    <div
      className="bg-black h-28 rounded-md"
    />
    <div
      className="bg-black h-28 rounded-md"
    />
    <div
      className="bg-black h-28 rounded-md"
    />
  </div>
</div>
`;
